/web/dataset/search_read
{"id": 267106619, "error": {"message": "Odoo Session Expired", "code": 100, "data": {"name": "odoo.http.SessionExpiredException", "debug": "Traceback (most recent call last):\n  File \"/usr/lib/python3/dist-packages/odoo/http.py\", line 653, in _handle_exception\n    return super(JsonRequest, self)._handle_exception(exception)\n  File \"/usr/lib/python3/dist-packages/odoo/http.py\", line 312, in _handle_exception\n    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])\n  File \"/usr/lib/python3/dist-packages/odoo/tools/pycompat.py\", line 87, in reraise\n    raise value\n  File \"/usr/lib/python3/dist-packages/odoo/addons/base/ir/ir_http.py\", line 197, in _dispatch\n    auth_method = cls._authenticate(func.routing[\"auth\"])\n  File \"/usr/lib/python3/dist-packages/odoo/addons/base/ir/ir_http.py\", line 117, in _authenticate\n    getattr(cls, \"_auth_method_%s\" % auth_method)()\n  File \"/usr/lib/python3/dist-packages/odoo/addons/base/ir/ir_http.py\", line 90, in _auth_method_user\n    raise http.SessionExpiredException(\"Session expired\")\nodoo.http.SessionExpiredException: Session expired\n", "exception_type": "internal_error", "message": "Session expired", "arguments": ["Session expired"]}}, "jsonrpc": "2.0"}
